### PR TITLE
[MB-1764] Added sample code and update docs.

### PR DESCRIPTION
**Pull request by hcrowell**
Added sample code and updated docs.

Android:
android_titanium_after

iOS:
ios_titanium_after

Testing:
- Tested on Android 6.0.1 Nexus 5X and iOS 9.3 iPod.

### CODE REVIEW

**Comment by rlepinski** [comment]
👍

File: example/app.js [hunk 1]
```diff
@@ -1,25 +1,229 @@
...
+
+UrbanAirship.addEventListener(UrbanAirship.EVENT_CHANNEL_UPDATED, function(e) {
+        Ti.API.info('Channel Updated: ' + UrbanAirship.channelId);
+        channelIdLabel.text = UrbanAirship.channelId;
+});
+
+UrbanAirship.addEventListener(UrbanAirship.EVENT_PUSH_RECEIVED, function(e) {
+        Ti.API.info('Push received: ' + e.message);
+        Ti.API.info('Extras: ' + JSON.stringify(e.extras));
+        labelMessage.text = UrbanAirship.channelId;
```
Comment (z0br0wn) [inline example/app.js]: should this be the message instead of the channel id?

Comment (hcrowell) [inline example/app.js]: good catch! i will update :)

**Comment by z0br0wn** [comment]
😎

**Pull request merged**